ui/app/AppLayouts/Chat/stores/MessageStore.qml

```diff
@@ -49,6 +49,28 @@ QtObject {
         return obj
     }
 
+    function getReplyMessageByIdAsJson(messageId: string) {
+        if (!messageModule) {
+            console.warn("getMessageByIdAsJson: Failed to parse message, because messageModule is not set")
```

Comment:
s/getMessageByIdAsJson/getReplyMessageByIdAsJson

Reply:
Same below

Reply:
Sorry, I don't get it :)

Reply:
The method name in `console.warn` is wrong (copy&pasted) from elsewhere I presume :)

Reply:
oh yes, sorry :D